app/models/upload.rb

```diff
@@ -542,6 +506,21 @@ def self.migrate_to_new_scheme(limit: nil)
     problems
   end
 
+  def self.extract_upload_ids(raw)
```

Comment:
This method of extracting uploads from Drafts and Profiles is much more strict than the old system, and will not detect all upload URL types.

Maybe an easy fix is to make the new regex more general? Rather than searching for upload://, we could search for a / followed by 40 consecutive hex characters. Or a / followed by 27 Base62 characters. That would be more similar to the existing logic. Any false-positives would be handled by the Upload.where(sha: sha1s) lookup?

(see https://dev.discourse.org/t/29848/38)

Reply:
I addressed this issue in the latest commit.